src/general/base_stage/CreatureStageHUDBase.cs

```diff
@@ -665,6 +671,19 @@ protected virtual void UpdateHealth(float delta)
         var hpText = StringUtils.FormatNumber(Mathf.Round(hp)) + " / " + StringUtils.FormatNumber(maxHP);
         hpLabel.Text = hpText;
         hpLabel.TooltipText = hpText;
+
+        var shaderMaterial = (ShaderMaterial)damageScreenEffect.Material;
+
+        if ((float)shaderMaterial.GetShaderParameter(fadeParameterName) > 0)
+        {
+            shaderMaterial.SetShaderParameter(fadeParameterName, (float)shaderMaterial
+                .GetShaderParameter(fadeParameterName) - delta);
```

Comment:
I think it would be excellent to hide the damage effect overlay when not necessary as it otherwise wastes quite a bit of GPU power (as fullscreen draw passes are pretty expensive) without any visible effect.